src/host.ps1

```diff
@@ -1,43 +1,73 @@
-$script:XO_HOST_FIELDS = "uuid,name_label,power_state,memory,address,hostname,version,productBrand"
+$script:XO_HOST_FIELDS = "id,uuid,name_label,name_description,power_state,memory,address,hostname,version,productBrand,build,cpus,startTime,tags,bios_strings,software_version,cpu_info,CPUs"
```

Comment:
Why both `id` and `uuid`, aren't they the same? Let's choose one and stick with it.